test/kata2.booleanToWord.test.js

```diff
@@ -1,5 +1,8 @@
 const { booleanToWord } = require("../src");
 
 describe("booleanToWord", () => {
-  // how do we create specs again???
+it('returns yes when passed to true', () => {
+    expect(booleanToWord(true)).toBe('yes')
+    expect(booleanToWord(false)).toBe('no')
+})
```

Comment:
Since youre inside a function here this should be indented:
```
  it('returns yes when passed to true', () => {
    expect(booleanToWord(true)).toBe('yes')
    expect(booleanToWord(false)).toBe('no')
  })
```